scripts/script.sh

```diff
@@ -1,25 +1,49 @@
 #!/usr/bin/env bash
+#!/usr/bin/env bash
+
+. "$(dirname "$0")/utils/script-utils.sh"
+. "$(dirname "$0")/utils/gh-utils.sh"
 
-. "$(dirname "$0")/utils.sh"
+check_env_var "REPO_ORG"
```

Comment:
Replace all occurrences of `REPO_ORG ` with `GITHUB_REPOSITORY_OWNER` because that's the GH syntax for it

```suggestion
```

Reply:
should now be resolved